dns.go

```diff
@@ -184,5 +184,18 @@ func getMapResponseDNSConfig(
 		dnsConfig = dnsConfigOrig
 	}
 
+	// Add device identifier to the nextdns resolvers, if there are any.
+	for _, resolver := range dnsConfig.Resolvers {
+		if strings.HasPrefix(resolver.Addr, "https://dns.nextdns.io/") {
+			resolver.Addr = fmt.Sprintf(
+				"%s?device_name=%s&device_model=%s&device_ip=%s",
+				resolver.Addr,
+				machine.GivenName,
+				machine.HostInfo.OS,
+				machine.IPAddresses[0],
+			)
```

Comment:
This just adds a nice machine identifier in the request log in the NextDNS dashboard.
![Machine identifier example](https://user-images.githubusercontent.com/1291396/200412718-d467c95e-2a30-48f5-af9c-c9ba8757c8d4.png)